docker/bundle.Dockerfile

```diff
@@ -29,6 +29,6 @@ LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 
 # Copy files to locations specified by labels.
-COPY bundle/manifests /manifests/
-COPY bundle/metadata /metadata/
-COPY bundle/tests/scorecard /tests/scorecard/
+COPY bundle/backstage.io/manifests /manifests/
```

Comment:
When I run `make bundle`, I get changes to this file, the default profile being `rhdh`:
```diff
diff --git a/bundle/rhdh/manifests/backstage-operator.clusterserviceversion.yaml b/bundle/rhdh/manifests/backstage-operator.clusterserviceversion.yaml
index 29da40cd..701b516b 100644
--- a/bundle/rhdh/manifests/backstage-operator.clusterserviceversion.yaml
+++ b/bundle/rhdh/manifests/backstage-operator.clusterserviceversion.yaml
@@ -29,7 +29,7 @@ metadata:
     categories: Developer Tools
     certified: "true"
     containerImage: registry-proxy.engineering.redhat.com/rh-osbs/rhdh-rhdh-rhel9-operator:1.3
-    createdAt: "2024-08-26T15:27:13Z"
+    createdAt: "2024-08-27T21:00:45Z"
     description: Red Hat Developer Hub is a Red Hat supported version of Backstage.
       It comes with pre-built plug-ins and configuration settings, supports use of
       an external database, and can help streamline the process of setting up a self-managed
diff --git a/docker/bundle.Dockerfile b/docker/bundle.Dockerfile
index af7df49d..496d61c1 100644
--- a/docker/bundle.Dockerfile
+++ b/docker/bundle.Dockerfile
@@ -29,6 +29,6 @@ LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
 LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
 
 # Copy files to locations specified by labels.
-COPY bundle/backstage.io/manifests /manifests/
-COPY bundle/backstage.io/metadata /metadata/
-COPY bundle/backstage.io/tests/scorecard /tests/scorecard/
+COPY bundle/rhdh/manifests /manifests/
+COPY bundle/rhdh/metadata /metadata/
+COPY bundle/rhdh/tests/scorecard /tests/scorecard/
```

Just like with the per-profile `bundle` dirs (https://github.com/redhat-developer/rhdh-operator/pull/62#discussion_r1731025145), I guess we need to have separate `docker/$PROFILE` directories too, each with their specific `bundle.Dockerfile`.
I guess we'll also need to update the `bundle-build` Makefile target accordingly as well, to be able to build the right bundle image for the given profile.

Reply:
moved to bundle/profile/*